build.gradle

```diff
@@ -30,12 +30,10 @@ jacocoTestReport {
 
 dependencies {
     implementation "commons-logging:commons-logging:1.1.1"
-    implementation "org.xerial.snappy:snappy-java:1.1.8.4"
+    implementation "org.xerial.snappy:snappy-java:1.1.10.1"
     implementation "org.apache.commons:commons-compress:1.22"
     implementation 'org.tukaani:xz:1.8'
-    implementation ('org.sharegov:mjson:1.4.1') {
-        exclude group: "junit"
-    }
+    implementation "org.json:json:20230227"
```

Comment:
Might as well go to the newest release?  20230618

Reply:
@lbergelson: must have just missed that when i wrote this. done.